app/workers/scheduled_publishing_worker.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class ScheduledPublishingWorker
+  include Sidekiq::Worker
+  sidekiq_options retry: 10
+  sidekiq_retry_in { 30 }
```

Comment:
These look good 👍 Might be worth leaving a comment above them about the maths of how they add up to 5 mins

Reply:
Done 👍